project/Dependencies.scala

```diff
@@ -17,19 +17,17 @@ object Dependencies {
   val sslConfigVersion = "0.2.1"
 
   val Versions = Seq(
-    crossScalaVersions := Seq("2.11.8"), // "2.12.0-RC2"
+    crossScalaVersions := Seq("2.11.8"), // "2.12.0"
```

Comment:
It was because we only released with best effort with those dependencies that were available for the milestone/rc.